src/SqlStreamStore.TestUtils/Postgres/PostgresDatabaseManager.cs

```diff
@@ -51,7 +51,7 @@ private async Task<bool> DatabaseExists(NpgsqlConnection connection, Cancellatio
 
             using(var command = new NpgsqlCommand(commandText, connection))
             {
-                return await command.ExecuteScalarAsync(cancellationToken).ConfigureAwait(false)
+                return await command.ExecuteScalarAsync(cancellationToken)
```

Comment:
I guess the reason you removed it is that it's in the tests?

Reply:
yes